jams/core.py

```diff
@@ -1048,26 +1163,50 @@ def to_html(self):
                             <th>value</th>
                             <th>confidence</th>
                         </tr>
-                    </thead>'''
+                    </thead>'''.format(self.namespace, n)
+
         out += r'''<tbody>'''
-        for i, obs in enumerate(self.data):
+
+        if max_rows is None or n <= max_rows:
+            out += self._fmt_rows(0, n)
+        else:
+            out += self._fmt_rows(0, max_rows//2)
+            out += r'''<tr>
+                            <th>...</th>
+                            <td>...</td>
+                            <td>...</td>
+                            <td>...</td>
+                            <td>...</td>
+                        </tr>'''
+            out += self._fmt_rows(n-max_rows//2, n)
+
+        out += r'''</tbody>'''
+
+        out += r'''</table></div>'''
+
+        out += r'''</div></div></div>'''
+        return out
+
+    def _fmt_rows(self, start, end):
+        out = ''
+        for i, obs in enumerate(self.data[start:end], start):
             out += r'''<tr>
                             <th>{:d}</th>
-                            <td>{:0.6f}</td>
-                            <td>{:0.6f}</td>
+                            <td>{:0.3f}</td>
```

Comment:
or a more general `jams.set_printoptions` ala pandas?  Maybe.  I guess we could add a global config dict for this sort of thing, but I wouldn't mind punting that downstream for a point release.

It would probably be best to add a rc/config file as well, which opens up a few more headaches than I'd like to deal with right now.